website/templates/emails/notify_base.mako

```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${website.settings.DOMAIN + "settings"}>user settings</a>. </p>
```

Comment:
Since you only use `settings` from `website`, it would be better to only import `settings` (instead of the entire website module)